src/components/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -7,11 +7,16 @@ import { pause } from 'utils/time';
 type Props = {
   textToCopy: string;
   children: ReactNode;
+  successText?: string;
 };
 
 const SECTION_DURATION_MS = 400;
 
-export default function CopyToClipboard({ textToCopy, children }: Props) {
+export default function CopyToClipboard({
+  textToCopy,
+  children,
+  successText = 'Link Copied',
```

Comment:
```suggestion
  successText = 'Copied link to clipboard,
```

Reply:
looks like the copy was updated a bit
![Screen Shot 2022-06-08 at 22 47 20](https://user-images.githubusercontent.com/80802871/172632753-c371d472-d6a3-4986-98dd-37a77e38e1ed.png)